crates/ruff/src/rules/pylint/rules/subprocess_run_without_check.rs

```diff
@@ -56,10 +62,21 @@ pub(crate) fn subprocess_run_without_check(checker: &mut Checker, call: &ast::Ex
         .is_some_and(|call_path| matches!(call_path.as_slice(), ["subprocess", "run"]))
     {
         if call.arguments.find_keyword("check").is_none() {
-            checker.diagnostics.push(Diagnostic::new(
-                SubprocessRunWithoutCheck,
-                call.func.range(),
-            ));
+            let mut diagnostic = Diagnostic::new(SubprocessRunWithoutCheck, call.func.range());
+            let text: &str = checker.locator().slice(call.range());
+            let ends_with_comma = text[..text.len() - 1].trim_end().ends_with(',');
```

Comment:
If I'm reading correctly, I think this will evaluate to `true` if there's a comment after the trailing comma, which would cause us to insert a second comma between the two arguments. What about: find the last argument in the call, find the comma that follows it, then insert `check=False` after the comma (with branches to handle the cases in which there are no arguments to the call (strange) and no trailing comma)? `remove_argument` has similar logic: https://github.com/astral-sh/ruff/blob/086e11087ff1695b2ef02c689419388527801701/crates/ruff/src/autofix/edits.rs#L88.

Reply:
(Per your note in the summary, I'm not too worried about getting the whitespace exactly right here, but we need to be careful with the commas.)